staging/src/k8s.io/externaljwt/apis/v1alpha1/api.proto

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// To regenerate api.pb.go run `hack/update-codegen.sh protobindings`
+syntax = "proto3";
+
+package v1alpha1;
+
+option go_package = "k8s.io/externaljwt/apis/v1alpha1";
+
+import "google/protobuf/timestamp.proto";
```

Comment:
hrmm... this is failing codegen when I run it locally:

```
$ hack/update-codegen.sh protobindings
+++ [1106 14:12:34] Generating protobufs for 71 targets
+++ [1106 14:13:03] Generating protobuf bindings for 9 targets
google/protobuf/timestamp.proto: File not found.
api.proto:24:1: Import "google/protobuf/timestamp.proto" was not found or had errors.
api.proto:77:5: "google.protobuf.Timestamp" is not defined.
!!! [1106 14:13:16] Call tree:
!!! [1106 14:13:16]  1: /Users/liggitt/go/src/k8s.io/kubernetes/hack/lib/protoc.sh:40 kube::protoc::protoc(...)
!!! [1106 14:13:16]  2: hack/_update-generated-proto-bindings-dockerized.sh:50 kube::protoc::generate_proto(...)
!!! [1106 14:13:16] Call tree:
!!! [1106 14:13:16]  1: /Users/liggitt/go/src/k8s.io/kubernetes/hack/lib/protoc.sh:40 kube::protoc::protoc(...)
!!! [1106 14:13:17]  2: hack/_update-generated-proto-bindings-dockerized.sh:50 kube::protoc::generate_proto(...)
!!! [1106 14:13:17] Call tree:
!!! [1106 14:13:17]  1: hack/update-codegen.sh:888 codegen::protobindings(...)
```

Reply:
I think we have to add https://github.com/protocolbuffers/protobuf/blob/v28.3/src/google/protobuf/timestamp.proto to third_party/protobuf/google/protobuf/timestamp.proto ... doing that made this generator work again

Reply:
Regenerated and worked for me